src/main/java/by/exadel/internship/entity/Internship.java

```diff
@@ -65,9 +66,14 @@
     @CollectionTable(name = "internship_skill", joinColumns = @JoinColumn(name = "is_inship_id"))
     private List<Skill> skills;
 
-//    private List<Form> formList;
-//    private List<User> techList;
-//    private List<User> adminList;
+    @ManyToMany(fetch = FetchType.LAZY)
+    @JoinTable(
+            name = "user_internship",
+            joinColumns = @JoinColumn(name = "ui_inship_id"),
+            inverseJoinColumns = @JoinColumn(name = "ui_u_id"))
+    @Type(type = "by.exadel.internship.mapper.enum_mapper.EnumTypePostgreSQL")
```

Comment:
Does @Type make sense here?

Reply:
now, i gues - no, but when we will conection users to internship use FE it will bee usefull.

Reply:
yaa i got it, thanks